data/themesongs.yml

```diff
@@ -1,35 +1,40 @@
 - title: 'Haircut'
   from: 'World Wrestling Federation'
   link: 'https://youtu.be/4JvNSdsQNqg'
-  embed: '<iframe width="560" height="315" src="https://www.youtube.com/embed/4JvNSdsQNqg" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>'
+  embed: '<iframe class="theme-song-card__emebed" src="https://www.youtube.com/embed/4JvNSdsQNqg" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>'
```

Comment:
coud you limit this `embed`  field to just the id or url 4JvNSdsQNqg or https://www.youtube.com/embed/4JvNSdsQNqg
and generate the rest of it in helpers?